proposals/add_requestTypes_tag_support

```diff
@@ -0,0 +1,54 @@
+# Kyverno Design Proposal - Adding Request Types to match and exclude
+
+Author: [Prateek Nandle](https://github.com/Prateeknandle)
+
+PR: [#3619](https://github.com/kyverno/kyverno/pull/3619)
+
+## Contents
+
+- [Kyverno Design Proposal - Adding Request Types to match and exclude](#kyverno-design-proposal---adding-request-types-to-match-and-exclude)
+  * [Introduction](#introduction)
+  * [Proposal](#proposal)
+  * [Example](#example)
+  * [Implementation](#implementation)
+
+## Introduction
+
+Until now we use `request.operation` under `condition.key`, to filter a request, after this proposal we'll use a tag `requestTypes` in `match` or `exclude` block to match/exclude specific requests coming from admission review requests.
```

Comment:
This would be `preconditions[]` as it could be in key or value.

Reply:
updated